code/modules/mob/living/simple_animal/abnormality/waw/apex_predator.dm

```diff
@@ -0,0 +1,189 @@
+	ranged = TRUE
+	melee_damage_lower = 30
+	melee_damage_upper = 40
+	move_to_delay = 3
+
+	melee_damage_type = RED_DAMAGE
+	armortype = RED_DAMAGE
+	stat_attack = HARD_CRIT
+
+	fear_level = 0	//You should never notice it
+
+	attack_sound = 'sound/abnormalities/fragment/attack.ogg'
+	attack_verb_continuous = "stabs"
+	attack_verb_simple = "stab"
+	faction = list("hostile")
+	can_breach = TRUE
+	threat_level = WAW_LEVEL
+	start_qliphoth = 2
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = list(0, 0, 35, 40, 45),
+						ABNORMALITY_WORK_INSIGHT = list(0, 0, 35, 40, 45),
+						ABNORMALITY_WORK_ATTACHMENT = list(25, 20, 15, 10, 0),
+						ABNORMALITY_WORK_REPRESSION = list(0, 0, 50, 55, 55)
+						)
+	work_damage_amount = 10
+	work_damage_type = RED_DAMAGE
+
+	ego_list = list(
+		/datum/ego_datum/weapon/animalism,
+		/datum/ego_datum/armor/animalism
+		)
+//	gift_type =  /datum/ego_gifts/animalism
+
+	var/revealed = TRUE
+	var/can_act = TRUE
+	var/backstab_damage = 200
+	var/agent_status //Used for insanity
+
+	var/jumping	//Used so it can only start one jump at once
+	var/busy	//Can we move now?
+
+	var/jump_cooldown
+	var/jump_cooldown_time = 5 SECONDS
+	var/jump_damage = 60
+
+
+
+/mob/living/simple_animal/hostile/abnormality/apex_predator/Move()
+	if(busy)
+		return FALSE
+	..()
+
+
+/mob/living/simple_animal/hostile/abnormality/apex_predator/NeutralEffect(mob/living/carbon/human/user, work_type, pe)
+	if(prob(40))
+		datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/apex_predator/FailureEffect(mob/living/carbon/human/user, work_type, pe)
+	datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/apex_predator/AttemptWork(mob/living/carbon/human/user, work_type)
+	if(user.health != user.maxHealth)
+		work_damage_amount = 20
+	return TRUE
+
+/mob/living/simple_animal/hostile/abnormality/apex_predator/PostWorkEffect(mob/living/carbon/human/user, work_type, pe, work_time)
+	if(user.health < 0)
+		datum_reference.qliphoth_change(-1)
+
+	work_damage_amount = initial(work_damage_amount)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/apex_predator/BreachEffect(mob/living/carbon/human/user)
+	..()
+	Cloak()
+	GiveTarget(user)
+
+/mob/living/simple_animal/hostile/abnormality/apex_predator/AttackingTarget()
+	if(!can_act)
+		return
+
+	if(!revealed)
+		//Will want this to be crazy
+		say("Behind you.")
+		FearEffect()
+
+		SLEEP_CHECK_DEATH(7)
+		Decloak()
+		SLEEP_CHECK_DEATH(3)
+
+		//Backstab
+		if(target in range(1,src))
+			visible_message("<span class='danger'>\The [src] rips out [target]'s guts!</span>")
+			new /obj/effect/gibspawner/generic(get_turf(target))
+			if(ishuman(target))
+				var/mob/living/H = target
+				H.apply_damage(backstab_damage, RED_DAMAGE, null, H.run_armor_check(null, RED_DAMAGE), spread_damage = TRUE)
+			else
+				FindTarget()
```

Comment:
This else statement I believe is redundant as FindTarget is called later regardless.